sdk/src/types/block/slot/index.rs

```diff
@@ -61,3 +70,51 @@ impl From<SlotIndex> for u64 {
 
 #[cfg(feature = "serde")]
 string_serde_impl!(SlotIndex);
+
+#[cfg(test)]
+mod test {
+    use crate::types::block::protocol::ProtocolParameters;
+
+    #[test]
+    fn to_from_timestamp() {
```

Comment:
this doesn't test `from`

Reply:
`protocol_params.slot_index(timestamp);` calls from